geojson_pydantic/geometries.py

```diff
@@ -22,22 +22,25 @@ class GeoInterfaceMixin:
     """Geo interface mixin class"""
 
     @property
-    def __geo_interface__(self):
+    def __geo_interface__(self) -> Dict[str, Any]:
         """GeoJSON-like protocol for geo-spatial (GIS) vector data."""
-        result = self.dict()
+        result = self.dict()  # type: ignore [attr-defined]
         if "bbox" in result and result["bbox"] is None:
             del result["bbox"]
         return result
```

Comment:
let's get #86 merged before this PR

Reply:
Yeah, knew that was coming so just did an ignore rather than anything fancy. Happy to wait  and then it's unnecessary.